packages/server/lib/controllers/connect/postSessions.integration.test.ts

```diff
@@ -62,10 +62,7 @@ describe(`POST ${endpoint}`, () => {
         expect(res.json).toStrictEqual({
```

Comment:
Should test name be changed up at [line 51](https://github.com/NangoHQ/nango/pull/3159/files#diff-58987026fb92134e7b129490ae17232672346cd946c11896a962e5bbd0b73d81R51)

Reply:
good catch